main.tf

```diff
@@ -126,6 +110,22 @@ resource "aws_s3_bucket_server_side_encryption_configuration" "cdn_bucket" {
   }
 }
 
+resource "aws_s3_bucket_lifecycle_configuration" "default" {
+  count = var.enable_origin_bucket_intelligent_tiering_lifecycle_rule != [] ? 1 : 0
```

Comment:
Isn't this a bool?

Reply:
snap, updated. thank uuu

Reply:
Dear @albertrdixon i addressed your comments, please kindly find a time in your over booked schedule to rereview. Thank you, your truly work mate Sako